src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationService.java

```diff
@@ -96,11 +97,12 @@ private static boolean isFullAdmission(ResponseType responseType) {
             .getDefendant()
             .getMobilePhone()
             .orElse(null));
+        map.put("isCompanyOrOrganisation", isCompanyOrOrganisation(response.getDefendant()));
```

Comment:
it is a general practice to put the key in all uppercase or lowercase. you might want to consider changing this.